### ISSUE TITLE
NTR: Cadmium high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8437]
New term request Cadmium high in blood (segal_220303190301):   UMLS: 0855872

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6000444
- label: Elevated blood cadmiun concentration
- parents: [Blood xenobiotic (HP:0410172)](https://hpo.jax.org/app/browse/term/HP:0410172)
- pmid: PMID:27486374
- def: The concentration of cadmium in the blood is above the upper limit of a guidance value for total blood mercury concentrations that implies possible injurious effects.
- comment: Cadmium is a toxic metal that can be released into the environment from human activities such as the use of cadmium containing batteries and phosphate fertilizers, incineration of electronic wastes and tobacco smoking.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8437
- synonyms:
